.gitignore

```diff
@@ -6,6 +6,7 @@ build
 test/app
 gpt.html
 gpt-each-bidder3.html
+nate_dev
```

Comment:
FYI: You can stick "personal" ignores in your `~/.gitignore_global`. It'll apply to all your git repos that way.

Great place for stuff like `.DS_Store` too.